ubuntu/16.04/usr/local/share/bootstrap/update_permissions.sh

```diff
@@ -24,14 +24,20 @@ function update_permissions() {
         CODE_OWNER="$APP_USER"
         export APP_USER
         export CODE_OWNER
-
         if [[ "$group" = UNKNOWN ]]; then
             APP_GROUP="$APP_USER"
             CODE_GROUP="$APP_GROUP"
             export APP_GROUP
             export CODE_GROUP
             groupadd --system --gid "$group_id" "$APP_GROUP" || groupmod -g "$group_id" "$APP_GROUP"
+        else
+            APP_GROUP="$group"
+            CODE_GROUP="$APP_GROUP"
+            export APP_GROUP
```

Comment:
There’s some duplication here that prI à my could have needed a small refactoring :)

Reply:
Can’t update on mobile... but « prl à my » = « probably » 😅